packages/core/package.json

```diff
@@ -53,9 +53,9 @@
     "@keystone-ui/tooltip": "^6.0.0",
     "@nodelib/fs.walk": "^1.2.8",
     "@preconstruct/next": "^4.0.0",
-    "@prisma/client": "3.14.0",
-    "@prisma/migrate": "3.14.0",
-    "@prisma/sdk": "3.14.0",
+    "@prisma/client": "4.1.0",
+    "@prisma/internals": "4.1.0",
+    "@prisma/migrate": "4.1.0",
```

Comment:
This should probably contain the latest prisma version `4.2.1`?

Reply:
@mmachatschek we will probably bump that before merging, but at the time of last change, `4.1.0` was the latest